RestSharp/Extensions/StringExtensions.cs

```diff
@@ -23,31 +23,29 @@
 using System.Text.RegularExpressions;
 using System.Web;
 
-namespace RestSharp.Extensions
-{
-    public static class StringExtensions
-    {
+namespace RestSharp.Extensions {
+    public static class StringExtensions {
```

Comment:
I appreciate your effort but changing the whole file format is rude to other people.